projects/Mallard/src/App.tsx

```diff
@@ -35,6 +41,9 @@ fetchCacheClear().then((weOk: boolean) => {
     }
 })
 
+// Capture the first screen event
+sendAppScreenEvent({ screenName: ScreenTracking.Issue })
```

Comment:
Does the Ophan tracking know/need to know which context it is in. For example local/beta/release etc. And if so, does this have any implications?

Reply:
It does - and this is in the Native tidy up ticket. We need a way of using either debug or prod. Im not sure of the implications, relying on @maxspencer for that.